src/buildParams.ts

```diff
@@ -7,6 +7,6 @@ export const enableReduxLogging = !isProd && true;
 export const baseUrls: Record<string, string> = {
   newm: isProd
     ? "https://staging-newm-server.herokuapp.com/"
-    : "https://staging-newm-server.herokuapp.com/",
+    : "https://garage.newm.io/",
```

Comment:
I made this change and the production Url change on my branch as part of the following task, https://app.clickup.com/t/3vc8t71. Not sure how this should be handled, possibly comment/link it to your commit/branch on the ticket and then my PR will add the production Url to finalize?

It's also a breaking change since developers will need to create new accounts? Not sure if that matters much with the stage of development we are in.

Reply:
I think we can both make the change, I will change the prod url to the studio one on my end. If there's a merge conflict we can take care of it when we get to it. 

As far as it being a breaking change, we should be okay to make the change without worrying about that, we add a comment on the `web-dev` channel that we have made this change and we should be good to go.